public/angular/js/controllers/incidents/report_incident_modal.js

```diff
@@ -106,12 +130,48 @@ angular.module('Aggie')
         });
     };
 
+    $scope.saveReport = function(report) {
+      Report.save({ id: report._id }, report, function() {
+      }, function() {
+        flash.setAlertNow("Sorry, but that report couldn't be saved for some reason");
```

Comment:
NP: Is it necessary to say "for some reason" ? Either we could state the reason or remove that part since it doesnt really add anything..up to you.

Reply:
Fixed in: https://github.com/TID-Lab/aggie/commit/556591795b099aa5367e620965e5268e40adac7a

Something I also found strange. This wasn't made by me and is messaging used throughout aggie. I shortened it.